src/web/lib/components/ThemeColorsEditor/index.js

```diff
@@ -2,26 +2,55 @@ import React from "react";
 import classnames from "classnames";
 import { SketchPicker } from "react-color";
 import onClickOutside from "react-onclickoutside";
-import { colorLabels, colorsWithAlpha } from "../../../../lib/constants";
+import { colorLabels, colorsWithAlpha, ESC } from "../../../../lib/constants";
 import { colorToCSS } from "../../../../lib/themes";
 import Metrics from "../../../../lib/metrics";
 
 import "./index.scss";
 
 class ThemeColorsEditor extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleKeyPress = this.handleKeyPress.bind(this);
+  }
+
+  handleClick(name) {
+    const { selectedColor, setSelectedColor } = this.props;
+    setSelectedColor({ name });
```

Comment:
Could make the next 4 lines simpler with a single `setSelectedColor` call like this:
```javascript
setSelectedColor({
  name: selectedColor === name ? null : name 
});
```